pkg/contract.go

```diff
@@ -42,25 +43,35 @@ var ErrContractNotFound = errors.New("contract not found")
 // ErrInvalidContractText is used when contract texts cannot be parsed or contain invalid values
 var ErrInvalidContractText = errors.New("invalid contract text")
 
+// StandardSignerAttributes defines the standard list of attributes used for a contract.
+// If SignerAttribute name starts with a dot '.', it uses the configured scheme manager
+var StandardSignerAttributes = []string{
+	".gemeente.personalData.fullname",
+	".gemeente.personalData.firstnames",
+	".gemeente.personalData.prefix",
+	".gemeente.personalData.familyname",
+	"pbdf.pbdf.email.email",
+}
+
 // EN:PractitionerLogin:v1 Contract
 var contracts = map[Language]map[ContractType]map[Version]*Contract{
 	"NL": {"BehandelaarLogin": {"v1": &Contract{
-		Type:                 "BehandelaarLogin",
-		Version:              "v1",
-		Language:             "NL",
-		SignerAttributes:     []string{"nuts.agb.agbcode"},
-		Template:             `NL:BehandelaarLogin:v1 Ondergetekende geeft toestemming aan {{acting_party}} om namens {{legal_entity}} en ondergetekende het Nuts netwerk te bevragen. Deze toestemming is geldig van {{valid_from}} tot {{valid_to}}.`,
-		TemplateAttributes:   []string{"acting_party", "legal_entity", "valid_from", "valid_to"},
-		Regexp:               `NL:BehandelaarLogin:v1 Ondergetekende geeft toestemming aan (.+) om namens (.+) en ondergetekende het Nuts netwerk te bevragen. Deze toestemming is geldig van (.+) tot (.+).`,
+		Type:               "BehandelaarLogin",
+		Version:            "v1",
+		Language:           "NL",
+		SignerAttributes:   StandardSignerAttributes,
```

Comment:
This is not backwards compatible and will make logins for the Zorginzage bolt insecure. So this must be a v2? When employee/employer relation is added to the (local) registry this can be used in Zorginzage as well.

Reply:
Also link it to an issue so we can track it when released and update the migration guide/implementation guide